app/pages/Auth/AuthPage.presenter/AuthPage.presenter.tsx

```diff
@@ -195,6 +207,7 @@ export const AuthPage: FC = (): JSX.Element => {
       }
       setAccountIds(status.accountIds);
     } catch (err) {
+      logError(err, 'app/pages/Auth/AuthPage.presenter/AuthPage.presenter.tsx:onClickUnlockWallet');
       console.log(err); // eslint-disable-line no-console
```

Comment:
do we still need to console.log err if we're rendering the snackbar error?

Reply:
Logging the actual error object is still helpful for debugging purposes, so I think its worth having it there for now. The logError method massages the error object a bit to get it to an easily displayable form, and some fields like the optional `cause` field off of the `errorOptions` attribute are not passed through.

The intent of this PR is to strike a balance between helping the developers figure out what might be going wrong, and not displaying an unattractive JSON object directly to the end user.